crates/re_space_view_spatial/src/contexts/transform_context.rs

```diff
@@ -71,32 +71,32 @@ impl SceneContextPart for TransformContext {
     fn populate(
         &mut self,
         ctx: &mut re_viewer_context::ViewerContext<'_>,
-        scene_query: &re_viewer_context::SceneQuery<'_>,
+        query: &re_viewer_context::ViewQuery<'_>,
         _space_view_state: &dyn re_viewer_context::SpaceViewState,
     ) {
         re_tracing::profile_function!();
 
         let entity_db = &ctx.store_db.entity_db;
         let time_ctrl = &ctx.rec_cfg.time_ctrl;
-        let entity_prop_map = scene_query.entity_props_map;
+        let entity_prop_map = query.entity_props_map;
 
-        self.space_origin = scene_query.space_origin.clone();
+        self.space_origin = query.space_origin.clone();
 
         // Find the entity path tree for the root.
-        let Some(mut current_tree) = &entity_db.tree.subtree(scene_query.space_origin) else {
+        let Some(mut current_tree) = &entity_db.tree.subtree(query.space_origin) else {
             // It seems the space path is not part of the object tree!
             // This happens frequently when the viewer remembers space views from a previous run that weren't shown yet.
             // Naturally, in this case we don't have any transforms yet.
             return;
         };
 
-        let query = time_ctrl.current_query();
+        let latest_at_query = time_ctrl.current_query();
```

Comment:
Does `current_query()` always return a `latest_at_query`? This semantic naming could be invalidated if that changes.

Reply:
right now that's what it does, but I suppose in the future this might change but this would very likely affect all callsites 🤔. But yeah this should probably be `time_query`!